crocks/List.spec.js

```diff
@@ -123,7 +123,7 @@ test('List value', t => {
 test('List toArray', t => {
   const a = List([ 'some-thing', ['else', 43], 34 ]).toArray()
 
-  t.same(a, [ 'some-thing', 'else', 43, 34 ], 'folds the wrapped array')
+  t.same(a, [ 'some-thing', ['else', 43], 34 ], 'provides the wrapped array')
```

Comment:
Is this a bad test because of the nested arrays, or is it a bad test because it was failing?  I like how this shows the preserving of structure [now that `toArray` is implemented correctly].